pydis_site/apps/resources/templatetags/get_category_icon.py

```diff
@@ -21,6 +21,7 @@
     "Paid": "fa-dollar-sign",
     "Podcast": "fa-microphone-alt",
     "Project Ideas": "fa-lightbulb-o",
+    "Security": "lock-alt",
```

Comment:
Can someone tell me what this does? I am unfamiliar. (Where do I check that this change worked?)

Reply:
They're the icons at the bottom right in the following example 
![bild](https://user-images.githubusercontent.com/1515135/178165737-128d57e3-f198-44d7-9a0d-97adc7c9001b.png)

To confirm it works, find a resource that has the security category and confirm that at the bottom right you see a lock icon next to the security category for that resource.

Reply:
Replaced with `fa-solid fa-lock` and confirmed it works 
<img width="724" alt="image" src="https://user-images.githubusercontent.com/75038675/178166402-a4c8ad8f-1d6f-484d-aacb-c162d5b06723.png">